docs/docs/recipes.md

```diff
@@ -343,6 +343,51 @@ export default () => (
 - [More on Using Styled Components](/docs/styled-components/)
 - [Egghead lesson](https://egghead.io/lessons/gatsby-style-gatsby-sites-with-styled-components)
 
+### Using CSS Modules
+
+#### Prerequisites
+
+- A [Gatsby site](/docs/quick-start/) with an index page component
+- An existing 
```

Comment:
I think you forgot to add a few more words here 😄

Reply:
Sorry for not getting to this earlier. Looks like I missed something here.

Reply:
No need to say sorry 😉  We thank you for the work you've done! Our users will be super happy about this! We appreciate the work! 🥇